fastchat/conversation.py

```diff
@@ -976,6 +976,50 @@ def get_conv_template(name: str) -> Conversation:
         stop_str=["###", "</s>", "[UNK]"],
     )
 )
+# AquilaChat2-34B default template
+# source: https://huggingface.co/BAAI/AquilaChat2-34B/blob/4608b75855334b93329a771aee03869dbf7d88cc/predict.py#L212
+register_conv_template(
+    Conversation(
+        name="aquila-legacy",
+        system_message="A chat between a curious human and an artificial intelligence assistant. "
+        "The assistant gives helpful, detailed, and polite answers to the human's questions.\n\n",
+        roles=("### Human: ", "### Assistant: ", "System"),
```

Comment:
I think third element is never used. We can delete them

Reply:
> I think third element is never used. We can delete them

Done.